packages/schema/src/config/app.ts

```diff
@@ -152,6 +152,16 @@ export default defineUntypedSchema({
      */
     pageTransition: false,
 
+    /**
+     * Default values for view transitions.
+     *
+     * This can be overridden with `definePageMeta` on an individual page.
+     * Only booleans are allowed.
+     * @see https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental
+     * @type {typeof import('../src/types/config').NuxtAppConfig['viewTransition']}
+     */
+    viewTransition: false,
```

Comment:
I think this should be default to `true` if the feature is enabled to avoid breaking changes for others projects that are already using `viewTransitionAPI`.

Reply:
I implemented this, but am not sure if I did it the correct way 😅